testdata/project-v3-config/PROJECT

```diff
@@ -33,6 +33,19 @@ resources:
   version: v1
   webhooks:
     conversion: true
+    hub: true
```

Comment:
Looking at the initial PR:

--hub v1 --spoke v2

So, I understand that the hub is the version that will converted to the spoke one 
In this way, it cannot be true or false, it must o have the the version instead.